app/frontend/src/features/auth/signup/Signup.tsx

```diff
@@ -1,33 +1,133 @@
+    width: "100%",
+    marginTop: theme.spacing(2),
+    marginLeft: "auto",
+    marginRight: "auto",
+    borderTop: "1px solid #333333",
+  },
+  facebookButton: {
+    width: "100%",
+    height: "40px",
+    borderRadius: `${theme.shape.borderRadius * 2}px`,
+    boxShadow: "0px 0px 5px rgba(0, 0, 0, 0.25)",
+    minHeight: `calc(calc(${theme.typography.button.lineHeight} * ${
+      theme.typography.button.fontSize
+    }) + ${theme.typography.pxToRem(12)})`, // from padding
+    color: "#ffffff",
+    fontWeight: 700,
+    backgroundColor: "#2d4486",
+    marginTop: theme.spacing(6),
+    marginBottom: theme.spacing(2),
+  },
+  googleButton: {
+    width: "100%",
+    height: "40px",
+    borderRadius: `${theme.shape.borderRadius * 2}px`,
+    boxShadow: "0px 0px 5px rgba(0, 0, 0, 0.25)",
+    minHeight: `calc(calc(${theme.typography.button.lineHeight} * ${
+      theme.typography.button.fontSize
+    }) + ${theme.typography.pxToRem(12)})`, // from padding
+    color: "#ffffff",
+    fontWeight: 700,
+    backgroundColor: "#d9472e",
+  },
+  agreement: {
+    textAlign: "center",
+  },
+  logIn: {
+    marginTop: "auto",
+  },
+  logInLink: {
+    textDecoration: "none",
+    color: theme.palette.secondary.main,
+    fontWeight: 700,
+  },
+}));
+
 export default function Signup() {
-  const { authState } = useAuthContext();
+  const { authState, authActions } = useAuthContext();
   const authenticated = authState.authenticated;
   const error = authState.error;
+  const classes = useStyles();
+
+  useEffect(() => {
+    authActions.clearError();
+  }, [authActions]);
 
   return (
     <>
       {authenticated && <Redirect to="/" />}
-      <PageTitle>Signup</PageTitle>
-      {error && <Alert severity="error">{error}</Alert>}
-
-      <Switch>
-        <Route exact path={`${signupRoute}`}>
-          <EmailForm />
-        </Route>
-
-        <Route path={`${signupRoute}/:urlToken?`}>
-          <CompleteSignupForm />
-        </Route>
-      </Switch>
+      <Box className={classes.backgroundImage}></Box>{" "}
+      {/* todo: extract to component */}
+      <Box className={classes.signupPage}>
+        <Switch>
+          <Route exact path={`${signupRoute}`}>
+            <AuthHeader>Let's get started!</AuthHeader>
+            {error && <Alert severity="error">{error}</Alert>}
+            <EmailForm />
+            {/* <Divider>Or</Divider>  not yet available: https://next.material-ui.com/components/dividers/ */}
+            {/* Hidden for beta: 
+            <Divider classes={{ root: classes.divider }} flexItem />
+            <MuiButton className={classes.facebookButton}>
+              Sign up with Facebook
+            </MuiButton>
+            <MuiButton className={classes.googleButton}>
+              Sign up with Google
+            </MuiButton>
+            */}
+            <Typography variant="body1" className={classes.agreement}>
+              By signing up, you agree with the T&Cs of using the platform and
```

Comment:
Once this is merged, make a bug saying we need to add in links here so we don't forget. Seems important!

Reply:
https://github.com/Couchers-org/couchers/issues/685